doc/ref_arch/kubernetes/chapters/chapter06.md

```diff
@@ -16,7 +16,9 @@ The following functionalities are recommended for securing kubernetes platform;
 - Image Signing
 - Role-base Access Control
 - Secret Managment
-- Platform multitenant security
+- Kubernetes Cluster Multi-tenancy security
+  - Tenant can be distinct teams or workload types (Develpoment or Production) within an organisation, each with a namespace
```

Comment:
@manamsamuel Can a tenant have multiple namespaces within a cluster?

Reply:
Yes, a tenant can have multiple namespaces within a cluster. However, namespace are not nested inside one another as each kubernetes resource can only be in one namespace.

Reply:
@manamsamuel Can you fix in text -- current text implies a single namespace. Thanks